### PR TITLE
Adding chobby wrapper download messages

### PR DESCRIPTION
**Pull request by DeinFreund**
I'd like to help implement #1826 , how can I see loopback messages in Chobby for debugging?

### CODE REVIEW

File: Shared/PlasmaShared/Utils.cs [hunk 1]
```diff
@@ -782,6 +782,30 @@ public static Task AsTask(this WaitHandle handle, TimeSpan timeout)
...
+        {
+            var tcs = new TaskCompletionSource<object>();
+            var expiration = DateTime.UtcNow.Add(updateInterval);
+            var registration = ThreadPool.RegisterWaitForSingleObject(handle, (state, timedOut) =>
+            {
+                var localTcs = (TaskCompletionSource<object>)state;
+                Task.Run(callback);
+                if (timedOut)
+                    if (DateTime.UtcNow > expiration) localTcs.TrySetCanceled();
+                else
```
Comment (sprunk) [inline Shared/PlasmaShared/Utils.cs]: This `else` will bind to the nested `if` but its indentation suggests the intention was otherwise.

Comment (DeinFreund) [reply Shared/PlasmaShared/Utils.cs]: Indeed. That's one more reason why I'd like to know how to test this.

**Comment by GoogleFrog** [comment]
Loopback messages probably appear in the infolog. If they don't then edit the loopback widget locally.

**Comment by DeinFreund** [comment]
Unless you've done any testing, keep in mind that this might not work at all.

**Comment by Licho1** [comment]
You made a PR without any tests at all?

**Comment by DeinFreund** [comment]
Well it compiles and runs, but it might not work as expected. I'll make sure it works later today if you didn't.

**Pull request merged**